### PR TITLE
Bug 1139308 - Disabling SIM input until gets verified r=alive

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[albertopq albertopq (albertopq)](https://github.com/albertopq) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0c834b105058)

File: apps/system/js/sim_lock_system_dialog.js [hunk 1]
```diff
@@ -386,7 +387,24 @@
     }
   };
 
+  SimLockSystemDialog.prototype.disable = function() {
```
Comment (alivedise) [inline apps/system/js/sim_lock_system_dialog.js]: nit: disableInput

File: apps/system/js/sim_lock_system_dialog.js [hunk 2]
```diff
@@ -386,7 +387,24 @@
...
+    this.pinInput.disabled = true;
+    this.pukInput.disabled = true;
+    this.xckInput.disabled = true;
+    this.newPinInput.disabled = true;
+    this.confirmPinInput.disabled = true;
+  };
+
+  SimLockSystemDialog.prototype.enable = function() {
```
Comment (alivedise) [inline apps/system/js/sim_lock_system_dialog.js]: nit: enableInput

**Comment by try-server-hook** [comment]
[albertopq albertopq (albertopq)](https://github.com/albertopq) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=3912f667c77a)

**Comment by albertopq** [comment]
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0f2bbf335241d834f6aee7a04f0fefdf4a5d7688

**Pull request merged**